WindowsServerDocs/remote/remote-access/web-application-proxy/Publishing-Applications-with-SharePoint,-Exchange-and-RDG.md

```diff
@@ -47,121 +47,121 @@ The following table describes the Exchange services that you can publish through
 
-        ```
+        ```PowerShell
         Set-RDSessionCollectionConfiguration -CollectionName "MyAppCollection" -CustomRdpProperty "pre-authentication server address:s: https://rdg.contoso.com/rdweb/`nrequire pre-authentication:i:1"
         ```
 
-    -   In Windows Server 2008 R2:  
+    -   In Windows Server 2008 R2:
 
-        1.  Log onto the Terminal Server with an account that has Administrator privileges.  
+        1.  Log onto the Terminal Server with an account that has Administrator privileges.
 
-        2.  Go to **Start** >**Administrative Tools** > **Terminal Services** > **TS RemoteApp Manager.**  
+        2.  Go to **Start** >**Administrative Tools** > **Terminal Services** > **TS RemoteApp Manager.**
 
-        3.  In the **Overview** pane of TS RemoteApp Manager, next to RDP Settings, click **Change**.  
+        3.  In the **Overview** pane of TS RemoteApp Manager, next to RDP Settings, click **Change**.
 
-        4.  On the **Custom RDP Settings** tab, type the following RDP settings into the Custom RDP settings box:  
+        4.  On the **Custom RDP Settings** tab, type the following RDP settings into the Custom RDP settings box:
 
-            `pre-authentication server address: s: https://externalfqdn/rdweb/`  
+            `pre-authentication server address: s: https://externalfqdn/rdweb/`
 
-            `require pre-authentication:i:1`  
+            `require pre-authentication:i:1`
 
-        5.  When you are done, click **Apply**.  
+        5.  When you are done, click **Apply**.
 
-            This tells the Collection server to include the custom RDP properties in the RDP files that are sent to clients. These tell the client that pre-authentication is required and to pass the cookies for the pre-authentication server address to the Remote Desktop Connection client (mstsc.exe) . This, in conjunction with disabling HttpOnly on the Web Application Proxy application, allows the Remote Desktop Connection client (mstsc.exe) to utilize the Web Application Proxy authentication cookie obtained through the browser.  
+            This tells the Collection server to include the custom RDP properties in the RDP files that are sent to clients. These tell the client that pre-authentication is required and to pass the cookies for the pre-authentication server address to the Remote Desktop Connection client (mstsc.exe) . This, in conjunction with disabling HttpOnly on the Web Application Proxy application, allows the Remote Desktop Connection client (mstsc.exe) to utilize the Web Application Proxy authentication cookie obtained through the browser.
 
-            For more information on RDP, see [Configuring the TS Gateway OTP Scenario](https://technet.microsoft.com/library/cc731249(v=ws.10).aspx).  
+            For more information on RDP, see [Configuring the TS Gateway OTP Scenario](https://docs.microsoft.com/en-gb/previous-versions/windows/it-pro/windows-server-2008-R2-and-2008/cc731249(v=ws.10)).
```

Comment:
@illfated  Please remove the locale information from the URL

Reply:
Thanks, will do. Must have been tired enough not to notice that detail before.